server/routes/documents/letters-list/letters-list.controller.js

```diff
@@ -382,4 +409,12 @@
     }).length;
   }
 
+  function areAllChecked(req) {
+    if (!req.session.documentsJurorsList.checkedJurors) {
+      req.session.documentsJurorsList.checkedJurors = [];
+    }
```

Comment:
here we could probable `return false;` inside this if statement to avoid the app trying to iterate over potentially thousands of records without the need for.

Reply:
Updated